tensorflow_datasets/scripts/benchmark_dataset.py

```diff
@@ -0,0 +1,48 @@
+r"""Benchmark dataset.
+
+python -m tensorflow_datasets.scripts.benchmark_dataset \
+  --dataset dataset_name
+
+"""
+import time
+
+from absl import app
+from absl import flags
+
+import tensorflow_datasets as tfds
+
+# DEFAULT_DATA_DIR = os.path.expanduser(os.path.join("~", "tensorflow_datasets"))
+
+EPOCHS = 1
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('dataset', None, 'Dataset name')
+flags.DEFINE_string(
+    'data_dir', None,
+    'Path to dataset directory (auto-computed). Dataset should already present')
+flags.DEFINE_integer('epochs', EPOCHS, 'No. of iterations over the dataset')
+
+
+def benchmark(dataset, data_dir, epochs):
```

Comment:
We want to compare different pipeline together, with different `batch_size`, `split='train[10%:]'`, `ds = ds.map`,... so the current signature seems too restrictive.

Reply:
We want to allow user to import the script to benchmark their own pipeline.